packages/tabs/src/Tabs.tsx

```diff
@@ -231,25 +231,31 @@ function Tabs({
     }
 
     const isTabSelected = index === selected;
-    const { disabled, onClick } = child.props;
+    const { disabled, onClick, onKeyDown, className, ...rest } = child.props;
 
     const tabProps = {
-      ...child.props,
-      disabled,
       as,
+      disabled,
```

Comment:
```suggestion
```

Reply:
I pulled out disabled from props and am now only spreading rest so if I don't pass it explicitly then it won't be passed